management/server/activity/sqlite/sqlite.go

```diff
@@ -74,15 +109,36 @@ func processResult(result *sql.Rows) ([]*activity.Event, error) {
 			}
 		}
 
-		events = append(events, &activity.Event{
+		if targetEmail != nil {
+			email, err := store.emailEncrypt.Decrypt(*targetEmail)
+			if err != nil {
+				log.Errorf("failed to decrypt data: %s", *targetEmail)
+				meta["email"] = ""
+			} else {
+				meta["email"] = email
+			}
+		}
+
+		event := &activity.Event{
 			Timestamp:   timestamp,
 			Activity:    operation,
 			ID:          uint64(id),
 			InitiatorID: initiator,
 			TargetID:    target,
 			AccountID:   account,
 			Meta:        meta,
-		})
+		}
+
+		if initiatorEmail != nil {
+			email, err := store.emailEncrypt.Decrypt(*initiatorEmail)
+			if err != nil {
+				log.Errorf("failed to decrypt data: %s", *initiatorEmail)
```

Comment:
I'm not very sure if login `initiatorEmail` is helpful as it may be cryptic. Maybe ID and AccountID are better?

Reply:
If the decryption fail then the encrypted data is the most interesting information. The AccountID and ID does not carry any relevant information but if it necessary we can figure it out based on the encrypted string from the DB.

Reply:
OK, I see your point. Also, I have another thought. Could we have a situation in which the encryption is broken in a way that the email information will be exposed? Database and logger backend may have different access tiers. In my understanding, the combination of ID and AccountID is unique to kind the broken record and minimize the exposure.

Reply:
I would do it but maybe I am too paranoid. @mlsmaycon what do you think?

Reply:
@pappz, the user ID will be best for this output. in the end we can map it back to the failed hash using it